src/nvim/move.c

```diff
@@ -892,7 +892,7 @@ void curs_columns(
       extra += 2;
     }
 
-    if (extra == 3 || plines < so * 2) {
+    if (extra == 3 || plines <= so * 2) {
```

Comment:
Mh, funny, this didn't trigger any errors on my machine. I guess this should actually be
```suggestion
    if (extra == 3 || plines < (so * 2 + ui_has(kUIMessages))) {
```
?

Reply:
Actually, no, this fix is correct event in the TUI.